data/net/minecraft/world/level/levelgen/synth/SurfaceNoise.mapping

```diff
@@ -1,5 +1,11 @@
 CLASS net/minecraft/world/level/levelgen/synth/SurfaceNoise
+	COMMENT The noise sampler used during surface building.
+	COMMENT This value is used as the {@code noise} parameter of {@link net.minecraft.world.level.levelgen.surfacebuilder.SurfaceBuilder#apply}.
+	COMMENT
+	COMMENT Mojang samples this noise rather strangely, by passing in (blockX, blockZ, 0.0625, localX).
```

Comment:
Rendered views won't pick up on new paragraphs unless you use the `<p>` tag. This is probably needed in a few other places too. 
```suggestion
	COMMENT <p>Mojang samples this noise rather strangely, by passing in (blockX, blockZ, 0.0625, localX).
```